tests/c/c_test.c

```diff
@@ -174,6 +174,75 @@ static int handles_init(handles_t *handles, const char *connstr)
   return -1;
 }
 
+static SQLSMALLINT sqltype_2_ctype(SQLLEN v, SQLLEN is_unsigned)
```

Comment:
why the geometry type not found on the list ?

Reply:
This function is a conversion of the SQL type to the SQL_C type, which is defined in sql.h/sqlext.h by standard ODBC and has no GEOMETRY type

Reply:
OK